components/boloforms/actions/common/base.mjs

```diff
@@ -0,0 +1,31 @@
+import { ConfigurationError } from "@pipedream/platform";
+import boloforms from "../../boloforms.app.mjs";
+
+export default {
+  props: {
+    boloforms,
+    subject: {
+      type: "string",
```

Comment:
I don't see `/actions/common/base.mjs` being used anywhere. I think it can be removed.